src/site/styles/index.md-njk

```diff
@@ -18,12 +18,11 @@ the style of buttons or adjust the thickness of borders on form inputs.
 
 ## Typography, spacing and layout
 
-The [MOD.UK Figma Design Library]({{ vars.figmaURLs.designLibrary }}) shows how
-to apply:
+Some of our guides are in the [MOD.UK Figma Design Library]({{ vars.figmaURLs.designLibrary }}) – the following links open in new tabs in Figma:
 
-- [typography]({{vars.figmaURLs.typography}})
-- [spacing]({{vars.figmaURLs.spacing}})
-- [layout]({{vars.figmaURLs.layout}})
+- <a rel="noreferrer noopener" target="_blank" href="{{ vars.figmaURLs.typography }}">typography <span class="govuk-visually-hidden">(opens in new tab)</span></a>
```

Comment:
Hmm actually not sure the space does do anything, but might be better inside the `<span>` if we have it as you can select the space if it's outside.